js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -730,12 +730,16 @@ var DropDownList = DropDownEditor.inherit({
         this._refreshPopupVisibility();
     },
 
+    _needToOpenPopup: function() {
```

Comment:
Use `_needToOpenPopup: this._hasItemsToShow.bind(this)`

Reply:
I suggest to rename it to something like `isNeedToOpenPopup`